application/data/install/schema.sql

```diff
@@ -185,11 +185,12 @@ CREATE TABLE `resource_template_property` (
   `alternate_label` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
   `alternate_comment` longtext COLLATE utf8mb4_unicode_ci,
   `position` int(11) DEFAULT NULL,
-  `data_type` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+  `data_type` varchar(766) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
```

Comment:
Pretty sure this is going to be a compatibility issue: not all MySQL installs will allow such a long value and associated index.

Reply:
I didn't check mysql, but it is not a limit for Travis. Anyway, the index can be removed, because it is not really used (a check is done in the adapter).

Reply:
This is the default limit of mysql: https://dev.mysql.com/doc/refman/5.6/en/innodb-limits.html, but I'm removing the constraint.